src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/GenericDefinitionEETypeNode.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using ILCompiler.DependencyAnalysisFramework;
```

Comment:
Nit: usings are usually grouped as: system/framework provided first, blank line, internal/ILCompiler ones next, blank line, then the rest that have `=` in them.